docs/snippets/solid/button-story-click-handler-args.js.mdx

```diff
@@ -0,0 +1,26 @@
+```js
```

Comment:
For snippets that contain JSX, we should use `jsx` for the language.

(I realize this is also incorrect for our React—and likely other–snippets. ☹️)